examples/observability/docker-compose.local-observability.yml

```diff
@@ -4,10 +4,23 @@ networks:
 
 services:
   zipkin:
-    image: openzipkin/zipkin
+    image: openzipkin/zipkin:latest
+    environment:
+      - STORAGE_TYPE=mysql
+      - MYSQL_HOST=zipkin-storage
+      - MYSQL_USER=zipkin
+      - MYSQL_PASS=zipkin
+      - JAVA_OPTS=-Xms1g -Xmx1g -XX:+ExitOnOutOfMemoryError
```

Comment:
Could we add a comment documenting what each of these does? If I'm reading it right we're increasing heap allocation and telling the jvm to exit on the first OOM instead of trying to soldier on, right?